src/modules/layout/components/HeaderNavigationBar.css

```diff
@@ -21,7 +21,12 @@
   text-transform: uppercase;
 }
 
-.navigation-item.active-navigation-item {
+.navigation-item.inactive-navigation-item {
+  opacity: 0.5;
+  cursor: default;
```

Comment:
You could also set user-select: none; in the Header.css in .header: This prevents that the user can select text (and therefore the cursor will be displayed as default).

Reply:
Done (/)